src/receivers/MCServer.hs

```diff
@@ -64,7 +64,7 @@ jsonURL :: String
 jsonURL = "https://api.mcsrvstat.us/2/"
 
 getJSON :: T.Text -> IO B.ByteString
-getJSON server_ip = simpleHttp $ jsonURL ++ (T.unpack server_ip)
+getJSON server_ip = simpleHttp $ jsonURL <> (T.unpack server_ip)
```

Comment:
This will face a compile-time error because both `jsonURL` and `T.unpack server_ip` is a String, while `<>` takes only T.Text as arguments.